src/test/e2e/api/proxy/proxy.e2e.test.ts

```diff
@@ -104,6 +104,37 @@ test('should not allow requests with a client token', async () => {
         .expect(403);
 });
 
+test('Should find a token by alias', async () => {
+    const tokenAlias = 'testclientkey';
+    await createApiToken(ApiTokenType.PROXY, {
+        alias: tokenAlias,
+    });
+
+    await createFeatureToggle({
+        name: 'enabledFeature1',
+        enabled: true,
+        strategies: [{ name: 'default', constraints: [], parameters: {} }],
+    });
+
+    await app.request
```

Comment:
Could test a request with the new key as well, just to make sure that both work.

Reply:
Doesn't this test also verify that, because the old key is just used to resolve the key with the new format?

Reply:
There is also a test verifying it here: https://github.com/Unleash/unleash/pull/1931/files/69b2b2729c73cabf41399e0f033e37c490e0359d#diff-0527b2389c8d5179e9b86200bd1533d12b2798b54da2d21490989e1cc1dc1cabR178

Reply:
What would be good to test is that the token retrieves from the correct project and environments.